bpm/bonita-core/bonita-process-engine/src/test/java/org/bonitasoft/engine/api/impl/ProcessAPIImplTest.java

```diff
@@ -1250,4 +1252,24 @@ SExpressionImpl createExpression(String name) {
         sExpression.setName(name);
         return sExpression;
     }
+
+    @Test
+    public void getDesignProcessDefinition_Should_Return_Design() throws Exception{
```

Comment:
Isn't this test the same as saying "API.getDesignProcessDefinition() should call service.getDesignProcessDefinition()" ?
In that case, it would only suppose you verify(service).getDesignProcessDefinition();